internal/obsreportconfig/obsmetrics/obsmetrics.go

```diff
@@ -7,6 +7,7 @@
 package obsmetrics // import "go.opentelemetry.io/collector/internal/obsreportconfig/obsmetrics"
 
 const (
-	NameSep = "/"
-	Scope   = "go.opentelemetry.io/collector/obsreport"
+	NameSep   = "/"
```

Comment:
I assume it's still being used for spans. Maybe it's worth renaming it to SpanSep for clarity?

Reply:
Maybe even `MetricNameSep` and `SpanNameSep`...

Reply:
correct, i can rename it

Reply:
done